src/components/NewbieSelect/NewbieSelect.tsx

```diff
@@ -5,25 +5,36 @@ import Box from '@material-ui/core/Box';
 import { QueryBuddyArgs, Query, Newbie } from 'buddy-app-schema';
 import AuthContext, { AuthContextData } from 'contexts/AuthContext';
 import { NEWBIE_SELECT } from 'graphql/newbie-select.graphql';
+import { makeStyles, Theme } from '@material-ui/core/styles';
 import PlusButton from 'components/PlusButton';
 import Carrousel from 'components/Carrousel';
 import PageContainer from 'components/PageContainer';
 import Header, { MenuTypes } from 'components/Header';
-import MenuContext from 'contexts/MenuContext';
+import MenuContext, { MenuContextData } from 'contexts/MenuContext';
 import NewbieSelectDictionary from './dictionary';
 
+const useStyles = makeStyles<Theme>(theme => ({
```

Comment:
This code is not necessary, we are applying styles exclusively  to `Box` component that already provide props to be used for this effect. Why over-engineer ?